### PR TITLE
[web] Fix 'lint' tasks and update documentation

### PR DESCRIPTION
**Pull request by imobachgs**
Fix the `lint` task to run `eslint` instead of `prettier` and update the documentation of the web frontend.

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -1,68 +1,27 @@
...
+This application offers a web-based UI which connects to [yastd](file:../yastd) through D-bus, using
+[Cockpit's infrastructure](https://cockpit-project.org/guide/latest/api-base1.html), and drives the
+installation process. It is built with the popular [React JavaScript library](https://reactjs.org/)
+and it was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Tasks
+
+Most of the tasks you can run through `npm` are provided by Create React App (see the
+[documentation](https://create-react-app.dev/docs/available-scripts/) for further details). However,
+we have added a few more tasks on our own. So let's recap all the available task (they are defined
```
Comment (dgdavid) [inline web/README.md:12]:
```suggestion
we have added a few more tasks on our own. So let's recap all the available tasks (they are defined
```

File: web/README.md [hunk 2]
```diff
@@ -1,68 +1,27 @@
...
+  opens the browser. The page will reload if change the source code.
+* `npm test`: launches the test runner in interactive watch mode. See [running
+  tests](https://create-react-app.dev/docs/running-tests) for more information.
+* `npm run build`: builds the application for production in the `build` folder.
+* `npm run lint`: runs [ESLint](https://eslint.org/) on the source code.
+* `npm run prettier`: runs [Prettier](https://prettier.io/) and modifies the code to stick to the
+  [rules](./.prettierrc).
+
+Additionally, there a special [`npm run
+eject`](https://create-react-app.dev/docs/available-scripts/#npm-run-eject) exists. It removes the
```
Comment (dgdavid) [inline web/README.md:25]: What about 

```suggestion
Additionally, an special [`npm run
eject`](https://create-react-app.dev/docs/available-scripts/#npm-run-eject) exists. It removes the
```

or 

```suggestion
Additionally, there is an special [`npm run
eject`](https://create-react-app.dev/docs/available-scripts/#npm-run-eject). It removes the
```

**Review by dgdavid — APPROVED** [review]
I left two minor comments. Other than that, it LGTM.

**Comment by imobachgs** [comment]
Thanks for the review! I have fixed the problems.

**Pull request merged**